src/fsharp/FSharp.Build/Fsc.fs

```diff
@@ -32,15 +32,15 @@ type FscCommandLineBuilder () =
     let builder = new CommandLineBuilder()
     let mutable args = []  // in reverse order
     let mutable srcs = []  // in reverse order
+
     /// Return a list of the arguments (with no quoting for the cmd.exe shell)
-    member x.CapturedArguments() =
-        List.rev args
+    member x.CapturedArguments() = List.rev args
```

Comment:
Possible additional cleanup: Whenever you see a mutable F# list plus a call to ``List.rev`` it's almost always cleaner to make it a use of a  ``ResizeArray`` (i.e. ``System.Collections.Generic.List<T>``) without the need for a reverse.  It's a common F# code smell